src/gui/UserStatusSelector.qml

```diff
@@ -134,7 +134,7 @@ ColumnLayout {
         TextField {
             id: userStatusMessageTextField
             Layout.fillWidth: true
-            placeholderText: qsTr("What is your Status?")
+            placeholderText: qsTr("What's your status?")
```

Comment:
I would need an English native speaker to confirm but for a tooltip I am convinced that `is` form is to be used.

Reply:
Well. @mgallien  is right. In English, the spelling of the questions and the official form is not abbreviated.
So we change to the version from "is" and in other applications I change.
I only leave the changed: "status".